.github/workflows/migrator-comment.yml

```diff
@@ -0,0 +1,32 @@
+name: Migrator comment in diff
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  COMMENT: 'generated by polaris-migrator DO NOT COPY'
+
+jobs:
+  check-diff:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+
+      - name: Check for generic migrator comment in diff
+        run: |
+          if git diff --name-only main...HEAD | grep -q '.*\.scss'; then
+            echo 'SCSS files were changed'
+          else
+            exit 0
+          fi
+          if git diff --unified=0 main...HEAD | grep "^\+.*$COMMENT"; then
```

Comment:
Would be nice if this only ran on diffs of changed SCSS files. Perhaps, there's a way to extract those above and use in both conditions. This is very slick though!

Reply:
Great point! I'll revise it to combine the conditions